catalog.html

```diff
@@ -55,6 +55,151 @@
+          </li>
+          <li>
+            <a class="catalog-breadcrumb-link" href="#">Каталог товаров</a>
+          </li>
+          <li>
+            <a class="catalog-breadcrumb-link" href="#">Моноподы для селфи</a>
+          </li>
+        </ul>
+      </section>
+      <section class="catalog-filter">
+        <h2>ФИЛЬТР:</h2>
+        <h3>Стоимость</h3>
+        <div class="slidecontainer">
+          <input type="range" min="1" max="#" value="50" class="slider" id="myRange">
+          <!-- здесь не поняла немного как выставить максимальное значение (на макете не вижу) и что подставлять в value-->
+        </div>
+        <h2>СОРТИРОВКА:</h2>
+        <button class="catalog-drop-button">Популярное</button>
+        <a class="catalog-button" href="#"><span class="visually-hidden">Вверх</span></a>
+        <a class="catalog-button" href="#"><span class="visually-hidden">Вниз</span></a>
+        <fieldset class="catalog-filter-group">
+          <legend class="catalog-filter-title">Цвет</legend>
+          <ul>
+            <li>
+              <label class="control">
+                <input class="control-input" type="checkbox" name="black" checked>
+                Черный
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label class="control">
+                <input class="control-input" type="checkbox" name="white" checked>
+                Белый
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label class="control">
+                <input class="control-input" type="checkbox" name="blue">
+                Синий
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label class="control">
+                <input class="control-input" type="checkbox" name="red">
+                Красный
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label class="control">
+                <input class="control-input" type="checkbox" name="pink">
+                Розовый
+              </label>
+            </li>
+          </ul>
+        </fieldset>
+        <fieldset class="catalog-filter-group">
+          <legend class="catalog-filter-title">Bluetooth</legend>
+          <ul class="catalog-filter-list">
+            <li class="catalog-filter-item">
+              <label class="control">
+                <input class="control-input" type="radio" name="bluetooth-group" value="yes" checked>
+                Есть
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label class="control">
+                <input class="control-input" type="radio" name="bluetooth-group" value="no">
+                Нет
+              </label>
+            </li>
+          </ul>
+        </fieldset>
+        <button class="button" type="button">Показать</button>
```

Comment:
Эта кнопка должна отправить нашу форму, поэтому у нее должен быть другой тип как раз:
```suggestion
        <button class="button" type="submit">Показать</button>
```